airbyte-config/models/src/main/resources/types/StandardNormalizationSummary.yaml

```diff
@@ -0,0 +1,15 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardNormalizationSummary.yaml
+title: StandardNormalizationSummary
```

Comment:
Thinking again about the naming here - the word `Standard` in `StandardSyncSummary` comes from the fact that we used to refer to a "connection" as a "standard sync". AFAIK we have never referred to normalization as "standard normalization", so I would prefer if we go through and rename all occurrences of `StandardNormalizationSummary` in this PR to `NormalizationSummary`